ncm-hostsfile/src/main/pan/components/hostsfile/config.pan

```diff
@@ -5,5 +5,13 @@
 
 unique template components/${project.artifactId}/config;
 
-include { 'components/${project.artifactId}/config-common' };
-include { 'components/${project.artifactId}/config-rpm' };
+include 'components/${project.artifactId}/schema';
+
+# Package to install
+"/software/packages" = pkg_repl("ncm-${project.artifactId}", "${no-snapshot-version}-${rpm.release}", "noarch");
+
+# Set prefix to root of component configuration.
+prefix '/software/components/${project.artifactId}';
+
+'active' ?= false;
+'dispatch' ?= false;
```

Comment:
Previously this was true (and for active). version has been dropped too: this is what causes ncm-cdispd to re-run the component when the rpm is updated.